v2/emailpassword/common-customizations/usermetadata/clear-data.mdx

```diff
@@ -0,0 +1,235 @@
+---
+id: clear-data
+title: Delete stored data
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./emailpassword/common-customizations/usermetadata/clear-data.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from "@theme/TabItem";
+import NpmOrScriptTabs from "/src/components/tabs/NpmOrScriptTabs";
+import NodeJSFrameworkSubTabs from "/src/components/tabs/NodeJSFrameworkSubTabs";
+
+# Deleted the stored data
```

Comment:
We should also talk about deleting a particular field from the metadata object separately. In nodejs, that would be by passing `null`, but in other SDKs, there would be a specific function to delete a particular field from the meta data object.